Miniforge3/construct.yaml

```
@@ -1,9 +1,9 @@
-{% set version = os.environ.get("MINIFORGE_VERSION", "24.11.3-0") %}
-{% set conda_libmamba_solver_version = "24.9.0"%}
-# when mamba_version is updated here, also update MICROMAMBA_VERSION in scripts/build.sh
-# As of Dec 2024 -- mamba 2.0.5 isn't compatible with constructor
-# https://github.com/conda-forge/miniforge/issues/697
-{% set mamba_version = "1.5.12" %}
+{% set version = os.environ.get("MINIFORGE_VERSION", "24.11.3-1") %}
```

Also, if `conda v25.1.0` is released with https://github.com/conda-forge/conda-feedstock/pull/256 being merged:

```suggestion
{% set version = os.environ.get("MINIFORGE_VERSION", "25.1.0-0") %}
```